include/irr/asset/IMeshPacker.h

```diff
@@ -160,6 +160,19 @@ class IMeshPacker : public MeshPackerBase
         return size;
     }
 
+public:
+    struct Triangle
+    {
+        uint32_t oldIndices[3];
+    };
+
+    struct TriangleBatch
+    {
+        core::vector<Triangle> triangles;
```

Comment:
far more performant is to have
```cpp
struct TriangleBatch
{
    uint16_t triangleCount = 0;
    Triangle triangles[MAX_BATCH_SIZE];
};
```
There's nothing I hate more than nested vectors!